packages/react-search-ui/src/types/Result.js

```diff
@@ -2,15 +2,10 @@ import PropTypes from "prop-types";
 
 import FieldValueWrapper from "./FieldValueWrapper";
 
-// Typically an object where keys are the field names, and values are field values.
-// Also could be literally any other arbitrary value depending on the particular Search API response.
-// Default views in Search UI know what to do with FieldValueWrapper values, but not arbitrary values, so it
-// is usually better to work with FieldValueWrapper values. We encourage FieldValueWrapper, but we accept
-// anything because we don't want users to have type error warnings unnecessarily.
+// If field is a "FieldValueWrapper", it can be displayed by the UI.
 //
-// An example would be if a user requests "grouping" in an App Search API request. That will come back
-// as "_group: {..}". It *should* be there in the Result so that a developer has it available to work
-// with.
```

Comment:
It's kind of a shame to lose all this context - the example in particular is helpful. Will we be adding it back at any point?

Reply:
I can add it back. I actually read it back to myself and I thought it sounded longwinded and confusing which is why I removed it.

Reply:
Totally up to you, I'm good either way!